provider/keycloak_openid_client_service_account_role.go

```diff
@@ -0,0 +1,113 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"strings"
+)
+
+func resourceKeycloakOpenidClientServiceAccountRole() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientServiceAccountRoleCreate,
+		Read:   resourceKeycloakOpenidClientServiceAccountRoleRead,
+		Delete: resourceKeycloakOpenidClientServiceAccountRoleDelete,
+		// This resource can be imported using {{realm}}/{{client_id}}. The Client ID is displayed in the GUI
```

Comment:
This comment doesn't seem correct.  You can just remove it if you want.

Reply:
Removed